frontend/src/exploration/questions/Questions.tsx

```diff
@@ -1,20 +1,122 @@
+import explorationQuestions from '../../stores/exploration/ExplorationQuestions';
+import { ExplorationAnswer } from '../../stores/exploration';
 
-const useStyle = makeStyles(() =>
+const useStyle = makeStyles((theme: Theme) =>
   createStyles({
-    container: {
-      backgroundColor: red[500], // TODO change/remove
-      height: '75vh', // TODO change/remove
+    heading: {
+      fontSize: theme.typography.pxToRem(15),
+      fontWeight: theme.typography.fontWeightRegular,
+    },
+    formControl: {
+      margin: theme.spacing(3),
     },
   })
 );
 
+// Interface to keep track of checked answers in component state.
+interface ExplorationSelection {
+  answer: ExplorationAnswer;
+  checked: boolean;
+}
+
+// Map answers to string made of questionIndex and answerIndex, in order to add and
+// remove them from the explorationStore in handleChange.
+const selections: Record<string, ExplorationSelection> = {};
+explorationQuestions.forEach((q, qIndex) => {
+  q.answers.forEach((a, aIndex) => {
+    selections[`explorationAnswer-${qIndex}-${aIndex}`] = {
+      answer: a,
+      checked: false,
+    };
+  });
+});
+
 function Questions(): JSX.Element {
   const classes = useStyle();
-  return <Box className={`${classes.container} Questions`}>TODO</Box>;
+  const explorationStore = useService(ExplorationStore);
+  const [state, setState] = useState(selections);
+
+  /**
+   * Handles change event of the exploration questions,
+   * by adding newly selected answers and
+   * removing de-selected answers from the ExplorationStore.
+   * @param event the changeEvent emitting this function.
+   */
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, checked } = event.target;
+    const { answer } = state[name];
+
+    setState({
+      ...state,
+      [name]: { answer, checked },
+    });
+
+    if (event.target.checked) {
+      explorationStore.addAnswer(answer);
+    } else {
+      explorationStore.removeAnswer(answer);
+    }
+  };
+
+  return (
+    <>
+      <Box>
+        {explorationQuestions.map((question, qIndex) => (
+          <Accordion>
+            <AccordionSummary
+              expandIcon={<ExpandMoreIcon />}
+              aria-controls={`question-${qIndex}-content`}
+              id={`question-${qIndex}-header`}
+            >
+              <Typography className={classes.heading}>
+                {question.text}
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails id={`question-${qIndex}-content`}>
+              <FormControl className={classes.formControl}>
+                <FormGroup>
+                  {question.answers.map((answer, aIndex) => (
+                    <FormControlLabel
+                      control={
+                        <Checkbox
+                          checked={
+                            state[`explorationAnswer-${qIndex}-${aIndex}`]
+                              .checked
+                          }
```

Comment:
```suggestion
                          checked={answers.some((a) => a === answer)}
```
Simply checks if answer is in selected answers from store